ui/component/livestreamComments/view.jsx

```diff
@@ -90,6 +113,22 @@ export default function LivestreamComments(props: Props) {
     }
   }
 
+  // Refresh timestamp on timer
+  React.useEffect(() => {
+    if (shouldRefreshTimestamp && (showTimestamps || isHovering)) {
+      const timer = setTimeout(() => {
+        setDateLastUpdate(Date.now());
+      }, UPDATE_TIMESTAMP_MS);
+
+      return () => clearTimeout(timer);
+    }
+  }, [shouldRefreshTimestamp, dateLastUpdate, showTimestamps, isHovering]);
+
+  // Refresh timestamp on initial hover, then if hovering leave to timer
+  React.useEffect(() => {
+    if (isHovering) setDateLastUpdate(Date.now());
+  }, [isHovering]);
+
```

Comment:
The hovering is causing too many updates, as shown in the video.  Can we skip this effect by changing `if (shouldRefreshTimestamp && (showTimestamps || isHovering))` to `if (shouldRefreshTimestamp)` so that the timer will always update it, and we don't need to do this "catch-up" effect.

Updating every 30s is probably less evil than updating on every mouse moving in and out?